core/storage/suggestion/gae_models.py

```diff
@@ -459,6 +461,32 @@ def get_suggestions_waiting_too_long_for_review(
             cls.last_updated
         ).fetch(MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_ADMIN)
 
+    @classmethod
+    def get_new_suggestions_waiting_for_review(
+        cls,
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Returns new suggestions waiting for review that were
+        submitted within timespan of SUGGESTION_REVIEW_WAIT_TIME_NOTIFICATION
+        days.
+
+        Returns:
+            list(GeneralSuggestionModel). A list of new suggestions
+            matching the criteria.
+        """
+        threshold_time = (
+            datetime.datetime.utcnow() - datetime.timedelta(
```

Comment:
Use the appropriate method from utils.py to get the current datetime

Reply:
In the codebase, I noticed this is how we are getting the current datetime, for e.g: https://github.com/oppia/oppia/pull/19045/files#diff-01f44d7b1b2d1be700ecfd1115d1dc9381f52404c1a334b0426e00e2f983d303R453. That's why, I used this way to get current time.

Reply:
@Ash-2k3 I think this got changed -- https://github.com/oppia/oppia/issues/18779. Please fix as needed.

Reply:
I meant this piece of code `https://github.com/oppia/oppia/blob/develop/core/storage/suggestion/gae_models.py#L451`

Reply:
Ah ok, thanks @Ash-2k3.

Could you still fix this to work with the changes in https://github.com/oppia/oppia/pull/18780 ? Given that utcnow has issues we probably shouldn't be using it.

Reply:
Done, I have tried a new approach can you please check if it's good ?